flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": ["x86_64"],
```

Comment:
this is disappointing

Reply:
Wanna do a PR at our repo to enable multi-arch builds?